src/components/ImageSegmentation/index.js

```diff
@@ -30,8 +24,7 @@ export default ({
   containerProps = emptyObj,
   onSaveTaskOutputItem,
 }) => {
-  const c = useStyles()
-  const [selectedIndex, changeSelectedIndex] = useState(0)
+  const [selectedIndex] = useState(0)
```

Comment:
it's problematic that the `selectedIndex` can't be changed. Out of scope for this PR